setup.py

```diff
@@ -26,7 +26,7 @@
     ],
     packages=find_namespace_packages(exclude=["tests"]),  # Required
     install_requires=[
-        "bioimageio.spec>=0.4.5",
+        "bioimageio.spec==0.4.8",
```

Comment:
I think that pin is too hard since it will only allow to install exact matches to `0.4.8`, but not `0.4.8post1` etc.
See https://www.darius.page/pipdev/ for checking pip dependencies.
I think `==0.4.8.*` should be fine.

Reply:
```suggestion
        "bioimageio.spec==0.4.8.*",
```